pycode/memilio-epidata/memilio/epidata_test/test_epidata_getCommuterMobility.py

```diff
@@ -316,6 +316,7 @@ def test_some_errors(self, mock_print):
         self.write_commuter_all_federal_states(self.path)
         self.assertEqual(len(os.listdir(self.path)), 17)
 
+    @unittest.skip
```

Comment:
Has this been referenced to be corrected anywhere?

Reply:
The `@unittest.skip` is currrently also on the main, because of the error in downloading the counties file. See Issue #310.